docker-compose.lb.override.yml

```diff
@@ -0,0 +1,11 @@
+version: '3'
+
+services:
+  lab-results-lb:
+    image: nginx:latest
+    volumes:
+      - ./nginx.conf:/etc/nginx/nginx.conf:ro
```

Comment:
Do we need the `nginx.conf` file from NHAIS?

Reply:
Proooobably?
Added